docs/recipes/docker-server/README.md

```diff
@@ -64,5 +64,17 @@ kubectl get service
 # kubernetes    ClusterIP      10.X.X.X       <none>         443/TCP        9m
 # lhci-server   LoadBalancer   10.X.X.X       X.X.X.X        80:XXXXX/TCP   2m
 ```
+### Docker-compose
+
+An example docker-compose.yml could look like this:
+```  lhci:
```

Comment:
is this a working docker-compose.yml? I think we're missing some parts like `version: `, `services: `, etc

Reply:
This is working in my setup, with the version do you mean the tag? That's in the image name. The service is not required for this container